examples/test-animate-color/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Animate Color</title>
+    <meta name="description" content="Animate Color - A-Frame">
+    <script src="../../dist/aframe.js"></script>
+  </head>
+  <body>
+    <a-scene stats="true">
+      <a-sky color="#afafaf"></a-sky>
+      <a-box color="white" position="2 1 -10">
```

Comment:
Is it possible to use the example you posted?

![](https://cloud.githubusercontent.com/assets/11075982/14265910/f15a7f64-fa89-11e5-9645-b33898705637.gif)